scilpy/utils/bvec_bval_tools.py

```diff
@@ -79,3 +81,94 @@ def check_b0_threshold(args, bvals_min):
 def get_shell_indices(bvals, shell, tol=10):
     return np.where(
         np.logical_and(bvals < shell + tol, bvals > shell - tol))[0]
+
+
+def fsl2mrtrix(fsl_bval_filename, fsl_bvec_filename, mrtrix_filename):
+    """
+    Convert a fsl dir_grad.bvec/.bval files to mrtrix encoding.b file.
+
+    Parameters
+    ----------
+    fsl_bval_filename: str
+        path to input fsl bval file.
+    fsl_bvec_filename: str
+        path to input fsl bvec file.
+    mrtrix_filename : str, optional
+        path to output mrtrix encoding.b file. Default is
+        fsl_bvec_filename.b.
+
+    Returns
+    -------
+
+    """
+    fsl_bval = np.loadtxt(fsl_bval_filename)
+    fsl_bvec = np.loadtxt(fsl_bvec_filename)
+
+    if not fsl_bvec.shape[0] == 3:
+        fsl_bvec = fsl_bvec.transpose()
+        logging.warning('WARNING: Your bvecs seem transposed. ' +
+                        'Transposing them.')
+
+    if not fsl_bval.shape[0] == 1:
```

Comment:
I'm not sure if this check is still valid. Maybe a change in numpy changed something, but my file is a valid bval file (on 1 line), but when loaded here it complains.